scripts/systemtap/kem/kemd.c

```diff
@@ -128,15 +127,15 @@ void kemd_rbs_free(struct kem_rb *rbs)
 int kemd_rbs_init(struct kem_rb *rbs)
 {
 	struct ke_msg *ptr;
-	int            i;
-	int            j;
+	int i;
+	int j;
 
 	for (i = 0; i < kemd_nr_cpus; i++) {
 		ptr = kzalloc(KEMD_BUFFER_SIZE * sizeof(struct ke_msg),
-				GFP_KERNEL);
+					  GFP_KERNEL);
 		if (ptr == NULL)
 			goto err;
-		rbs[i].kr_buf  = ptr;
+		rbs[i].kr_buf = ptr;
```

Comment:
The extra space is *intentional* added to align the '=' to the next line.

Reply:
nothing looking odd in source code, we will come to know after checking in